README.md

```diff
@@ -1,13 +1,31 @@
 # PassKit
 
 [![Swift Package Manager compatible](https://img.shields.io/badge/SPM-compatible-brightgreen.svg)](https://github.com/apple/swift-package-manager)
-[![Platform](https://img.shields.io/badge/Platforms-macOS%20|%20Linux-lightgrey.svg)](https://github.com/gargoylesoft/PassKit)
 
-A Vapor package which handles all the server side elements required to implement passes for Apple Wallet.
+🎟️ A Vapor package which handles all the server side elements required to implement passes for Apple Wallet.
 
-## NOTE
+### Major Releases
 
-This package requires Vapor 4.
+The table below shows a list of PassKit major releases alongside their compatible Swift versions. 
+
+|Version|Swift|SPM|
+|---|---|---|
+|0.2.0|5.9+|`from: "0.2.0"`|
+|0.1.0|5.9+|`from: "0.1.0"`|
+
+Use the SPM string to easily include the dependendency in your `Package.swift` file
+
+```swift
+.package(url: "https://github.com/vapor-community/PassKit.git", from: "0.2.0")
```

Comment:
I'm assuming this version will be 0.3.0? The README should be updated to reflect that